packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -59,6 +58,7 @@ class MediaQueryData {
     this.size = Size.zero,
     this.devicePixelRatio = 1.0,
     this.textScaleFactor = 1.0,
+    this.platformBrightness = ui.Brightness.light,
     this.padding = EdgeInsets.zero,
     this.viewInsets = EdgeInsets.zero,
     this.alwaysUse24HourFormat = false,
```

Comment:
@goderbauer, while you're here :D

I left a #27311. Null values for these fields don't really make much sense.

Reply:
ACK